### ISSUE TITLE
Tcp/UdpPort can't have value > 32768

### ISSUE THREAD

**Issue by EaseTheWorld** [kaitoy/pcap4j#81]
Because value type is (signed) Short
tcp/udp port is unsigned 16bit number but java's Short can't handle it.
As a result, some operation like compareTo() doesn't work.

**Comment by kaitoy** [comment]
Both UdpPort and TcpPort can represent values over 32768.
But, right, `compareTo()` doesn't behave naturally. I will fix it (and other named numbers, too).
What do you mean "some operation"? Are there other operations that don't work?

**Comment by EaseTheWorld** [comment]
Something about overflow.
I first found this issue when I tried UdpPort.getInstance(Short.parseShort("60000")).
If you change the type bigger than Short, it will be solved.

**Comment by kaitoy** [comment]
Do `UdpPort.getInstance((short) 60000)` instead.

**Comment by EaseTheWorld** [comment]
You are right. UdpPort.getInstance((short)Integer.parseInt(str)); will get it work at the moment.

**Comment by hyperxpro** [comment]
Couldn't it be nice if we replace short with int? @kaitoy

**Comment by kaitoy** [comment]
I should've created classes for unsigned numbers.
I don't have the motivation to do it for now, though.

**Comment by jonahmurphy** [comment]
This manifests in other places also, for example TcpHeader src/dst ports are broken.
